src/patterns.rs

```diff
@@ -74,8 +74,8 @@ impl Pattern {
 }
 
 #[derive(Debug, Default)]
-/// The **parsed equation** at a structural level: extracted constraints + collected forms
-/// + a solver-friendly order. Forms here are still raw strings; tokenize each with
+/// The **parsed equation** at a structural level: extracted constraints + collected forms +
+/// a solver-friendly order. Forms here are still raw strings; tokenize each with
```

Comment:
o/w the pretty-printed version thinks that `+ a solver...` is an (unordered) list item (thanks to starting with `+`)...